metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -265,3 +265,14 @@ spec:
         floor((ocs_storage_provider_operator_version>0)/1000) - ignoring(storage_consumer_name) group_right() floor((ocs_storage_client_operator_version>0)/1000) > 1 or floor((ocs_storage_client_operator_version>0)/1000) - ignoring(storage_consumer_name) group_left() floor((ocs_storage_provider_operator_version>0)/1000) >= 1
       labels:
         severity: critical
+  - name: ceph-perf-warnings.rules
+    rules:
+    - alert: OSDHighCPULoad
+      annotations:
+        description: High CPU usage in the OSD container on node {{ $labels.pod }}. Please create more OSDs to increase performance
+        message: High CPU usage detected in OSD container on node {{ $labels.pod}}.
+        severity_level: warning
```

Comment:
please remember to add the new "runbook_url" annotation with the link to the alert doc file.

Reply:
This can be a follow-up PR, since this is a new requirement.